### PR TITLE
Scroll to search result in preview

### PR DESCRIPTION
**Pull request by enahum**

### CODE REVIEW

File: app/components/post_list/post_list.js [hunk 1]
```diff
@@ -154,12 +155,17 @@ export default class PostList extends PureComponent {
         } = this.props;
         const {managedConfig} = this.state;
 
+        const highlight = highlightPostId && highlightPostId === postId;
```
Comment (jarredwitt) [inline app/components/post_list/post_list.js]: Why do we need to check for existence and equality? Wouldn't equality just be enough?

Comment (enahum) [reply app/components/post_list/post_list.js]: That was added by HH I just moved it to use it elsewhere

File: app/screens/search/search.js [hunk 2]
```diff
@@ -73,11 +74,17 @@ class Search extends PureComponent {
...
             postId: null,
             preview: false,
-            value: ''
+            value: '',
+            managedConfig: {}
         };
     }
 
+    componentWillMount() {
+        mattermostManaged.addEventListener('change', this.setManagedConfig);
```
Comment (jarredwitt) [inline app/screens/search/search.js]: Event listener needs to be removed on unmount right?

**Comment by enahum** [comment]
Done

File: app/screens/search/search.js (hunk 2 above)
Comment (enahum) [reply app/screens/search/search.js]: done, also removed it from the post_list were you added it before

File: app/screens/search/search.js [hunk 3]
```diff
@@ -99,6 +106,10 @@ class Search extends PureComponent {
         }
     }
 
+    componentWillUnmount() {
+        mattermostManaged.clearListeners();
```
Comment (hmhealey) [inline app/screens/search/search.js]: Doesn't this clear all listeners? So if you open the app (to the post list), go to the search view, and then close the search view, it'll unmount the post list's listener?

Comment (enahum) [reply app/screens/search/search.js]: actually you are totally right, the thing is that we don't have a way to clear only one listener, I'll make the change to include that as well in this ticket

**Comment by enahum** [comment]
@hmhealey @jarredwitt done

**Comment by jarredwitt** [comment]
@hmhealey good catch. Looks good to me, I had already approved it 😝

**Pull request merged**